pom.xml

```diff
@@ -59,6 +59,7 @@
     <findbugs.maxRank>9</findbugs.maxRank>
     <hadoop.version>2.6.3</hadoop.version>
     <logback.version>1.1.3</logback.version>
+    <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
```

Comment:
I think it is just STDOUT, not STDERR, like @keith-turner said.
I think converting to the log is also appropriate, but I also think doing this is useful, too. It helps separate the test output from the maven plugin output, so you can distinguish between the two. You can still use temporary print statements to debug, but with this, you'll be able to find that output in a file, separate from the plugin output.